c2corg_ui/templates/outing/view.html

```diff
@@ -127,7 +127,7 @@ <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" d
   % if not version:
   <div class="view-details-associations col-xs-12  associations">
     <span class="lead">
-      <div ng-show="userCtrl.auth.isAuthenticated()" class="add-association">
+      <div ng-show="userCtrl.auth.isAuthenticated() && userCtrl.hasEditRights('outings', ${ {"users": dumps(outing['associations']['users']) }})">
```

Comment:
Concerning the ``hasEditRights`` here, please see issue https://github.com/c2corg/v6_ui/issues/1036